src/upnp_mrreg.cc

```diff
@@ -160,6 +161,6 @@ void MRRegistrarService::prcoessSubscriptionUpdate(std::string sourceProtocol_CS
             config->getOption(CFG_SERVER_UDN).c_str(),
             serviceID.c_str(), event);
 
```

Comment:
Same here, this is passed into `genaNotifyAllCommon` eventually, which will free it. Should be freed on error though.